pages/linux/pidof.md

```diff
@@ -0,0 +1,19 @@
+# pidof
+
+> Gets the id of a process using it's name.
+
+- List all process ids with given name:
+
+`pidof {{bash}}`
+
+- List a single process id with given name:
+
+`pidof -s {{bash}}`
+
+- List process ids including scripts with given name:
+
+`pidof -x {{script.py}}`
+
+- Kill all processes with given name:
+
+``kill `pidof {{name}}` ``
```

Comment:
The single backtick syntax here isn't really good practice on the command line any more. Instead, use the `$(command)` syntax instead:
```suggestion
`kill "$(pidof {{name}})" `
```